Song_sort_Genre.php

```diff
@@ -42,7 +73,10 @@
 			<h3>Genre</h3>
 			<h3>Duration</h3>
 			<h3>Size</h3>				
+		</div>
+		<div class="content_4">
 			<?php
+				//Called up all data to display in the page and information of each column is place underneath their title
```

Comment:
21/9: This code below  use to called up the data and display it in the div box. First It called up the hosting server.